src/main/java/io/jenkins/plugins/synopsys/security/scan/extension/pipeline/SecurityScanStep.java

```diff
@@ -628,6 +718,15 @@ public ListBoxModel doFillProductItems() {
             }
             return items;
         }
+
+        @SuppressWarnings({"lgtm[jenkins/no-permission-check]", "lgtm[jenkins/csrf]"})
+        public ListBoxModel doFillPolaris_assessment_modeItems() {
+            ListBoxModel items = new ListBoxModel();
+            items.add(new ListBoxModel.Option("", ""));
```

Comment:
Check the other dropdowns, the first key should be "Select"

`items.add(new ListBoxModel.Option("Select", ""));`